src/components/Login.vue

```diff
@@ -15,31 +15,23 @@
           <p class="site-description">{{ $t('login.subheader') }}</p>
       </div>
       <md-layout md-align="center" md-gutter="16">
-      <md-layout md-flex="66" md-flex-xsmall="80">
-          <md-input-container class="password_input">
+          <LoginWithUserPassword v-if="userPassword" ref="login_with_user_password"></LoginWithUserPassword>
+          <md-layout v-else md-flex="66" md-flex-xsmall="80">
+            <md-input-container class="password_input">
               <label style="text-align: center;width: 100%;">{{ $t('login.password_label') }}</label>
               <md-input v-model="passPhrase" type="password" autocomplete="new-password" @keyup.native="kp($event)"></md-input>
```

Comment:
Why do we need such autocomplete value?

Reply:
Well, it is previuos implementation. If need to remove it, so let's do it